File: JavaScript/6 kyu/Are they the same.js
function comp(array1, array2){
  //your code here
  if (!array1 || !array2) {
    return false;
  } else {
    array1 = array1.map(a => { return a*a }).sort((a,b) => { return b-a }); 
    array2 = array2.sort((a,b) => { return b-a });
    for (let i = 0; i < array1.length; i++) {
      if (array2[i] !== array1[i]) {
         return false;
      }
    }
    return true;
  }
}

File: Python/6 kyu/Are they the -same--.py
def comp(array1, array2):
    # your code
    if array1 and array2 or array1 == [] and array2 ==[]:
        arr1 = sorted([pow(n, 2) for n in array1])
        arr2 = sorted(array2)
        for i in range(len(arr1)):
            if arr1[i] != arr2[i]:
                return False
        return True
    return False